client/src/ui/organisms/placement/index.tsx

```diff
@@ -114,24 +128,76 @@ export function TopPlacement() {
           extraClassNames={["top", "container"]}
           cta={placementData.top?.cta}
           imageHeight={50}
+          renderer={RenderSideOrTopBanner}
         ></PlacementInner>
       )}
     </div>
   );
 }
 
+export function HpMainPlacement() {
+  const placementData = usePlacement();
+  return HpPlacement({
+    placementData: placementData?.hpMain,
+    imageWidth: 970,
+    imageHeight: 250,
+  });
+}
+
+export function HpFooterPlacement() {
+  const placementData = usePlacement();
+  return HpPlacement({
+    placementData: placementData?.hpFooter,
+    imageWidth: 728,
+    imageHeight: 90,
+  });
+}
+
+function HpPlacement({
+  placementData,
+  imageWidth,
+  imageHeight,
+}: {
+  placementData?: PlacementData;
+  imageWidth: number;
+  imageHeight: number;
+}) {
+  const { backgroundColor } = placementData?.colors || {};
+  const css = Object.fromEntries(
+    [["--place-hp-main-background", backgroundColor]].filter(([_, v]) =>
```

Comment:
nit: similarly, could this be more generic and be something like `--place-hp-banner-background`